### PR TITLE
Use prettier-stylelint-formatter fork

### PR DESCRIPTION
**Pull request by ismail-syed**
* Replaces [`prettier-stylelint`](https://github.com/hugomrdias/prettier-stylelint) with a [fork](https://github.com/ismail-syed/prettier-stylelint-formatter) that addresses an [issue](https://github.com/hugomrdias/prettier-stylelint/issues/3)

cc/ @kaelig

### CODE REVIEW

**Comment by ismail-syed** [comment]
Yah 😢 
The verbose warnings are creating friction in our tooling.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+- Replaces [`prettier-stylelint`](https://github.com/hugomrdias/prettier-stylelint) with a [forked](https://github.com/ismail-syed/prettier-stylelint-formatter) version adressing an [issue](https://github.com/hugomrdias/prettier-stylelint/issues/3) [#23](https://github.com/Shopify/stylelint-config-shopify/pull/23)
```
Comment (kaelig) [inline CHANGELOG.md]: adressing -> addressing

**Comment by kaelig** [comment]
There's movement on the issue https://github.com/hugomrdias/prettier-stylelint/issues/3

@ismail-syed do you have cycles to subscribe to the issue and revert back to the normal package once it's solved?

**Comment by ismail-syed** [comment]
Sure, if it ever gets solved.

**Pull request merged**